google-cloud-firestore/lib/google/cloud/firestore/service.rb

```diff
@@ -48,7 +50,7 @@ def firestore
               config.endpoint = host if host
               config.lib_name = "gccl"
               config.lib_version = Google::Cloud::Firestore::VERSION
-              config.metadata = { "google-cloud-resource-prefix": "projects/#{@project}/databases/(default)" }
+              config.metadata = { "google-cloud-resource-prefix": "projects/#{@project}/databases/#{@database}" }
```

Comment:
What if `@database` is nil?

Reply:
Error will be thrown in this case. But `@database` can be nil only if the user explicity initialize an object of `Service` class, which is marked as private, and use it for sending requests. 
While initializing the firestore client, `@database` will have the the value `(default)` unless overridden by the user.